packages/blockchain-extension/extension/commands/startFabricRuntime.ts

```diff
@@ -52,15 +59,15 @@ export async function startFabricRuntime(registryEntry?: FabricEnvironmentRegist
             if (runtime instanceof LocalEnvironment) {
                 const isCreated: boolean = await runtime.isCreated();
                 if (!isCreated) {
-                    await runtime.create();
+                    await runtime.create(runtime.numberOfOrgs);
```

Comment:
why do we need to pass in the number of orgs here surely this property is set in the runtime?

Reply:
I think you're right!
I believe this was fixed when I decided to store the number of orgs in the environment entry.